test/functional/ex_cmds/bang_filter_spec.lua

```diff
@@ -28,7 +26,17 @@ describe('issues', function()
   end)
 
   it('#3269 Last line of shell output is not truncated', function()
-    command([[nnoremap <silent>\l :!ls bang_filter_spec<cr>]])
+    local map = [[nnoremap <silent>\l :!ls bang_filter_spec<cr>]]
+    local result = [[
+  :!ls bang_filter_spec                                |
+  ]]
+    if helpers.iswin() then
+      map = [[nnoremap <silent>\l :!dir /b bang_filter_spec<cr>]]
+      result = [[
+  :!dir /b bang_filter_spec                            |
+  ]]
```

Comment:
Can't the whitespace be done once below, instead of twice here? Also a ternary is clearer:

```lua
command(iswin() and [[nnoremap <silent>\l :!dir /b bang_filter_spec<cr>]]
                 or [[nnoremap <silent>\l :!ls bang_filter_spec<cr>]])
local result = (iswin() and [[:!dir /b bang_filter_spec                            |]]
                         or [[:!ls bang_filter_spec                                |]])
```